doc/RecordingControl.xml

```diff
@@ -2009,6 +2009,14 @@ secfrac = "." 1*6DIGIT</programlisting>
         </mediaobject>
       </figure>
     </section>
+    <section xml:id="_refCloudRecordingUpload">
+      <title>Uploading to an Object Storage</title>
+      <para>
+        A device signaling support for recording to an external target shall limit the upload speed so that a segment takes at least half as long as its duration to upload.
+        If the upload fails on the first try, the device shall retry uploading to the external target as fast as possible, to prevent loss of recording.
```

Comment:
do we have ONVIF API to control the segment upload speed against segment size & duration? Or this PR is about only recording control doc update.

Reply:
At the moment, only update to the documentation. Some other use cases were raised during the F2F and the proposal will have to be reviewed and adapted.

Reply:
Yes, we still need to make some updates to it, but we haven't had the time.

Reply:
@jflevesque-genetec thanks for the response I am sure it will be discussed and reviewed at the F2F.